src/app/core/components/widgets/widget-network/widget-network.component.ts

```diff
@@ -103,7 +112,8 @@ export class WidgetNetworkComponent extends WidgetComponent implements AfterView
   };
 
   constructor(
-    public router: Router, private ws: WebSocketService, private locale: LocaleService,
+    public router: Router, private ws: WebSocketService,
+    private locale: LocaleService, private rs: ReportsService,
```

Comment:
It's better not to abbreviate ReportsService to `rs`, it's not a common service and you won't be able to tell what it is if you were to see 'rs' in code.